LittlePoops/little-poops-01.py

```diff
@@ -0,0 +1,35 @@
+# Write a function that takes a positive integer as an argument &
+# returns a Boolean indicating whether the argument is a power of three.
+
+# Hint: you might want to use recursion for this, which we haven't talked about yet. Exciting!
+
+def threepower(x,p):
```

Comment:
It looks like you're checking for values that are cubes of some number. What we're looking for is a function to check whether the value is some exponential value of 3. 

So it should return `True` for 3, 9, 27, 81, ...